srcpkgs/talloc/template

```diff
@@ -1,32 +1,67 @@
 # Template file for 'talloc'
 pkgname=talloc
 version=2.3.1
-revision=1
-build_style=configure
-hostmakedepends="pkg-config python3 libxslt docbook-xsl which"
+revision=2
+build_style=waf3
+build_helper="qemu"
+configure_script="buildtools/bin/waf"
+configure_args="--sysconfdir=/etc --localstatedir=/var
+ --disable-rpath --disable-rpath-install --without-gettext
+ --builtin-libraries=replace --bundled-libraries=NONE"
+hostmakedepends="pkg-config docbook2x"
+makedepends="python3-devel libtirpc-devel libxslt gettext-devel"
 short_desc="Hierarchical pool based memory allocator with destructors"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
 homepage="https://talloc.samba.org/"
 distfiles="http://samba.org/ftp/${pkgname}/${pkgname}-${version}.tar.gz"
 checksum=ef4822d2fdafd2be8e0cabc3ec3c806ae29b8268e932c5e9a4cd5585f37f9f77
 
-do_configure() {
+if [ "$CROSS_BUILD" ]; then
+	configure_args+=" --cross-compile --hostcc=cc
+	 --cross-execute=qemu-${XBPS_TARGET_QEMU_MACHINE}-static"
+fi
+
+post_extract() {
+	# wrap target python3.9-config to find the right python flags
 	if [ "$CROSS_BUILD" ]; then
-		cp "$FILESDIR"/cross-32bit.answers .
-		_args+=" --cross-compile --hostcc=${CC} --cross-answers=cross-32bit.answers"
+		local _pyconfig="${XBPS_WRAPPERDIR}/python${py3_ver}-config"
+		cat> "${_pyconfig}" <<-EOF
+		#!/bin/sh
+		exec "${XBPS_CROSS_BASE}/usr/bin/python${py3_ver}-config" "\$@"
+		EOF
+		chmod 755 "${_pyconfig}"
 	fi
```

Comment:
Seeing as you used this in multiple places, maybe make into a build helper that adds it? (Does that even work?)

Reply:
If we change the default wrapper, there is no need for a build helper. I could go either way on whether the better wrapper is the right thing in general or I should just replicate it three times for Samba.